templates/referent/organizational_chart_macros.html.twig

```diff
@@ -1,39 +1,47 @@
-{% macro orga_chart_row(items, referent, editable) %}
+{% macro orga_chart_row(items, referent, editable, admin) %}
     {% import _self as oc %}
     <ol{% if items is not empty and items|first.level <= 2 %} class="organizational-chart"{% endif %}>
         {% for item in items %}
             <li>
-                {{ oc.orga_chart_item(item, referent, editable) }}
+                {{ oc.orga_chart_item(item, referent, editable, admin) }}
             </li>
         {% endfor %}
     </ol>
 {% endmacro %}
 
-{% macro orga_chart_item(item, referent, editable) %}
+{% macro orga_chart_item(item, referent, editable, admin) %}
     {% import _self as oc %}
     <div>
         <h{{ item.level }}>
```

Comment:
are you check the range of level `[1-6]`?

Reply:
no :s